README.rst

```diff
@@ -525,6 +525,7 @@ Please submit a pull request if you wish to include your company on the list:
 - `blk.io <https://blk.io>`_
 - `comitFS <http://www.comitfs.com/>`_
 - `ConsenSys <https://consensys.net/>`_
+- `TrustWallet <http://trustwalletapp.com>`
```

Comment:
Hi - you're missing the underscore at the end of the URL.

Please can you ensure your entry is in alphabetical order in the list.

Additionally, there's a companies rst file located under docs/source/companies.rst. Please could you include there (appreciate I don't make that clear).

Reply:
sure, thanks for review